src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -97,6 +98,16 @@ public void setCartFilePath_validPath_setsCartFilePath() {
         assertEquals(path, modelManager.getCartFilePath());
     }
 
+    @Test
+    public void emptyCart() {
+        modelManager.addCartIngredient(TypicalIngredients.ALMOND);
+        modelManager.addCartIngredient(TypicalIngredients.APPLE);
+        modelManager.cartMoveIngredients();
+        assertEquals(modelManager.getCart(), new Cart());
+        assertTrue(modelManager.hasInventoryIngredient(TypicalIngredients.ALMOND));
+        assertTrue(modelManager.hasInventoryIngredient(TypicalIngredients.APPLE));
```

Comment:
This is the main testing in `ModelManagerTest`. It checks to ensure that after the command, the cart is empty and the previously added ingredients are moved from the Cart to inventory.